mkosi/__init__.py

```diff
@@ -3080,6 +3080,28 @@ def reset_random_seed(root: Path) -> None:
     with complete_step("Removing random seed"):
         random_seed.unlink()
 
+HASHED_PASSWORD_RE = r'''
```

Comment:
```suggestion
HASHED_PASSWORD_RE = re.compile(r'''
```
This way the `re.VERBOSE` flag can live directly on the same object as the regex that is written in its style and setting the flag doesn't need to be remembered at the site of use.